### PR TITLE
Feature async

### PR DESCRIPTION
**Pull request by kimothokamau**
Add the following: 
- API to add user and score
- API handler for refreshing user scores

### CODE REVIEW

File: src/api-backend.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return response;
+};
+
+const getScores = async () => {
+  const refreshScores = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/zeWhEaKsT4ZEAaPGExUD/scores/');
+  const refreshScoresJson = refreshScores.json();
+  return refreshScoresJson;
+};
+
+export { postScores, getScores };
```
Comment (DeliceLydia) [inline src/api-backend.js]: - [ ] When I refresh to get a changed list a new list comes with all the score that was submitted before check here
rep

Normally when you refresh it has to show an updated list with just the score you just added currently it keeps repeating itself..please fix that!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @kimothokamau , 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -32,4 +32,29 @@ refreshBtn.addEventListener('click', () => {
...
+        const playerScoreElem = document.createElement('td');
+        playerScoreElem.innerHTML = `${playerscore.score}`;
+        scoreEntryElem.appendChild(playerScoreElem);
+        displayScores.appendChild(scoreEntryElem);
+      })
+    })
+}
+
 gameID('foorball');
+window.onload(loadScores());
```
Comment (henatan99) [inline src/index.js]: - [x] For some reason there are errors preventing the app from working properly. Kindly fix them.
- [x] The live demo is also not working and it has a different background. I don't know if you haven't updated it. 
Screenshot from 2021-08-06 22-57-41       
Screenshot from 2021-08-06 22-54-32

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @kimothokamau , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (kimothokamau) [reply src/index.js]: I have resolved the 2 errors mentioned:
- app not opening
- live link

File: src/api-backend.js (hunk 1 above)
Comment (kimothokamau) [reply src/api-backend.js]: I have resolved

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @kimothokamau 😃, 

Great job working on the requested changes. 💯
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**